modules/common/deps.sh

```diff
@@ -51,8 +51,17 @@ else
         info "CF cli version:"
         cf version
     fi
-fi
 
+    if [ ! -e "bin/eksctl" ]; then
+	curl --silent --location \
```

Comment:
This should go into https://github.com/SUSE/catapult/blob/master/backend/eks/deps.sh, that way it doesn't get downloaded for all backends, and the download is skipped with `DOWNLOAD_CATAPULT_DEPS=false` as in CI and catapult unit/integration tests.

Also, would you like to add a couple of lines in `catapult/tests/{unit,integration}-tests.sh` for `eksctl` :D? One can look at the existing ones grepping for `DOWNLOAD_CATAPULT_DEPS` there.

Reply:
I will try to remember for future works on new tools needed by catapult.